src/client/Microsoft.Identity.Client.Broker/BrokerExtension.cs

```diff
@@ -23,6 +24,25 @@ public static class BrokerExtension
         /// If a broker does not exist or cannot be used, MSAL will fallback to a browser.
         /// Make sure browser auth is enabled (e.g. if using system browser, register the "http://localhost" redirect URI, etc.)
         /// </remarks>
+        public static PublicClientApplicationBuilder WithWindowsBroker(this PublicClientApplicationBuilder builder, bool enableBroker = true)
```

Comment:
Yeah, I think we want to add `WithMacBroker` and `WithLinuxBroker` later. The danger of putting them under a single API is that you'll enable the broker, it'll default to browser on Mac, then we add support for Mac, developer picks it up without testing, and that'll break them because redirect URI for Mac isn't setup.

Reply:
+1 to Bogdan. I provided the same answer in the API design doc.